src/index.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+module.exports = {
+  Stop: require('./circuit/hop'),
+  Dialer: require('./dialer'),
+  multicodec: require('./multicodec'),
+  tag: 'Circuit'
+}
```

Comment:
-------------------------

Ok, following your lead on this, I can see some benefits if:

- Relay addrs are also supposed to be announced in things like DHT and/or Identify
- The peer that wants to be relayed (through user config on adding a Relayed multiaddr to its Addresses.Swarm) could check if the Relay peer is actually within is the reach (ping and check if it supports the Relay protocol), so that it is safe to announce that Relay addr to other peers.